# German translation for duplicity
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the duplicity package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2020-03-07 09:49-0600\n"
"PO-Revision-Date: 2018-02-23 13:10+0000\n"
"Last-Translator: Tobias Bannert <Unknown>\n"
"Language-Team: German <de@li.org>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2020-03-18 06:31+0000\n"
"X-Generator: Launchpad (build 3a6db24bbe7280ec09bae73384238390fcc98ad3)\n"

#: ../bin/duplicity:120
msgid "INT intercepted...exiting."
msgstr "INT abgefangen … wird beendet."

#: ../bin/duplicity:128
#, python-format
msgid "GPG error detail: %s"
msgstr "GPG-Fehlerdetail: %s"

#: ../bin/duplicity:138
#, python-format
msgid "User error detail: %s"
msgstr "Benutzerfehlerdetail: %s"

#: ../bin/duplicity:148
#, python-format
msgid "Backend error detail: %s"
msgstr "Fehlerdetail des Hintergrundprogramms: %s"

#: ../bin/rdiffdir:64 ../duplicity/commandline.py:236
#, python-format
msgid "Error opening file %s"
msgstr "Fehler beim Öffnen der Datei »%s«"

#: ../bin/rdiffdir:131
#, python-format
msgid "File %s already exists, will not overwrite."
msgstr "Datei »%s« besteht bereits; wird nicht überschrieben."

#: ../duplicity/asyncscheduler.py:70
#, python-format
msgid "instantiating at concurrency %d"
msgstr "Instanz wird mit Gleichzeitigkeit %d erstellt"

#: ../duplicity/asyncscheduler.py:97
msgid "inserting barrier"
msgstr "Sperre wird eingefügt"

#: ../duplicity/asyncscheduler.py:146
msgid "running task synchronously (asynchronicity disabled)"
msgstr "Aufgabe wird synchron ausgeführt (asynchronität deaktiviert)"

#: ../duplicity/asyncscheduler.py:152
msgid "scheduling task for asynchronous execution"
msgstr "Aufgabe für asynchrone Ausführung wird geplant"

#: ../duplicity/asyncscheduler.py:181
msgid "task completed successfully"
msgstr "Aufgabe erfolgreich abgeschlossen"

#: ../duplicity/asyncscheduler.py:192
msgid ""
"a previously scheduled task has failed; propagating the result immediately"
msgstr ""
"Eine vorherig geplante Aufgabe ist fehlgeschlagen; Ergebnis wird sofort "
"verbreitet"

#: ../duplicity/asyncscheduler.py:215 ../duplicity/asyncscheduler.py:236
#, python-format
msgid "active workers = %d"
msgstr "Aktive Arbeiter = %d"

#: ../duplicity/asyncscheduler.py:256
#, python-format
msgid "task execution done (success: %s)"
msgstr "Aufgabenausführung abgeschlossen (erfolgreich: %s)"

#: ../duplicity/backend.py:104
#, python-format
msgid "Import of %s %s"
msgstr "Import von %s %s"

#: ../duplicity/backend.py:213
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Hintergrundprogramm konnte nicht initialisiert werden: %s"

#: ../duplicity/backend.py:392
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Rückverfolgung des vorangegangenen Fehlers: %s"

#: ../duplicity/backend.py:408
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Nach %s Versuchen wird aufgegeben. %s: %s"

#: ../duplicity/backend.py:412
#, python-format
msgid "Attempt %s failed. %s: %s"
msgstr "Versuch %s ist fehlgeschlagen. %s: %s"

#: ../duplicity/backend.py:506
#, python-format
msgid "Reading results of '%s'"
msgstr "Ergebnisse von »%s« werden gelesen"

#: ../duplicity/backend.py:532
#, python-format
msgid "Writing %s"
msgstr "%s wird geschrieben"

#: ../duplicity/backend.py:573
#, python-format
msgid "File %s not found locally after get from backend"
msgstr ""
"Datei %s konnte nicht lokal gefunden werden, nach Erhalt vom "
"Hintergrundprogramm."

#: ../duplicity/backends/giobackend.py:109
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Verbindung fehlgeschlagen, bitte Passwort prüfen: %s"

#: ../duplicity/backends/multibackend.py:97
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend: Konnte den angefragten String %s nicht auswerten: %s "

#: ../duplicity/backends/multibackend.py:106
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr ""
"MultiBackend: angefragter String %s ist nicht gültig: mehr als einen Wert "
"für %s"

#: ../duplicity/backends/multibackend.py:111
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr ""
"MultiBackend: angefragter String %s ist nicht gültig: unbekannter Parameter "
"%s"

#: ../duplicity/backends/multibackend.py:161
#: ../duplicity/backends/multibackend.py:166
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "MultiBackend: nicht erlaubter Wert für %s: %s"

#: ../duplicity/backends/multibackend.py:177
#, python-format
msgid "MultiBackend: Url %s"
msgstr "MehrfachHintergrundprogramm: Adresse %s"

#: ../duplicity/backends/multibackend.py:181
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr ""
"MehrfachHintergrundprogramm: Konfigurationsdatei konnte nicht geladen werden "
"%s: %s "

#: ../duplicity/backends/multibackend.py:190
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MehrfachHintergrundprogramm: Speicher %s benutzen"

#: ../duplicity/backends/multibackend.py:195
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "MehrfachHintergrundprogramm: env %s = %s einstellen"

#: ../duplicity/backends/multibackend.py:208
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "Multibackend: Registeraffinität für Präfix %s"

#: ../duplicity/backends/multibackend.py:249
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MehrfachHintergrundprogramm: _put: Zum Speicher schreiben #%s (%s)"

#: ../duplicity/backends/multibackend.py:262
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr ""
"MehrfachHintergrundprogramm: Es konnte nicht zum Speicher geschrieben werden "
"#%s (%s), bitte #%s versuchen, Ausnahmefall: %s"

#: ../duplicity/backends/multibackend.py:269
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr ""
"MehrfachHintergrundprogramm: Schreiben von %s ist fehlgeschlagen. Vorgang "
"wird abgebrochen."

#: ../duplicity/backends/multibackend.py:276
#, python-format
msgid ""
"MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr ""
"MehrfachHintergrundprogramm: %s konnte nicht geschrieben werden. Es wurden "
"alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/multibackend.py:295
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht zu %s von %s gelangen"

#: ../duplicity/backends/multibackend.py:298
#, python-format
msgid ""
"MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr ""
"MehrfachHintergrundprogramm: %s konnte nicht erhalten werden. Es wurden alle "
"Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/multibackend.py:308
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr "MultiBackend: %s: %d Dateien"

#: ../duplicity/backends/multibackend.py:311
#, python-format
msgid "Exception during list of %s: %s"
msgstr "Ausnahme während der Liste von %s: %s"

#: ../duplicity/backends/multibackend.py:318
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MehrfachHintergrundprogramm: kombinierte Liste: %s"

#: ../duplicity/backends/multibackend.py:346
#, python-format
msgid "MultiBackend: failed to delete %s from %s"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht von %s gelöscht werden"

#: ../duplicity/backends/multibackend.py:350
#, python-format
msgid ""
"MultiBackend: failed to delete %s. Tried all backing stores and none "
"succeeded"
msgstr ""
"MehrfachHintergrundprogramm: %s konnte nicht gelöscht werden. Es wurden alle "
"Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/pydrivebackend.py:150
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "PyDrive-Hintergrundprogramm: mehrere Dateien aufgerufen »%s«."

#: ../duplicity/backends/webdavbackend.py:66
msgid "Missing socket or ssl python modules."
msgstr "Es fehlt ein Socket oder ein SSL Python Modul."

#: ../duplicity/backends/webdavbackend.py:84
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "Die Cacert Datenbankdatei '%s' ist nicht lesbar."

#: ../duplicity/backends/webdavbackend.py:105
msgid ""
"Option '--ssl-cacert-path' is not supported with python 2.7.8 and below."
msgstr ""
"Die Option '--ssl-cacert-path' wird von Python 2.7.8 oder früher nicht "
"unterstützt."

#: ../duplicity/backends/webdavbackend.py:109
#, python-format
msgid ""
"For certificate verification with python 2.7.8 or earlier a cacert database\n"
"file is needed in one of these locations: %s\n"
"Hints:\n"
"Consult the man page, chapter 'SSL Certificate Verification'.\n"
"Consider using the options --ssl-cacert-file, --ssl-no-check-certificate ."
msgstr ""
"Für die Zertifikatsprüfung mit Python 2.7.8 oder früher wird eine cacert "
"Datenbank\n"
"Datei an einem der folgenden Orte benötigt: %s\n"
"Hinweise:\n"
"Konsultieren Sie die Hilfeseiten, Kapitel 'SSL-Zertifikatsverifizierung'.\n"
"Erwägen Sie die Verwendung der Optionen --ssl-cacert-file, --ssl-no-check-"
"certificate ."

#: ../duplicity/backends/webdavbackend.py:155
#, python-format
msgid "Using WebDAV protocol %s"
msgstr "WebDAV-Protokol %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:156
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "WebDAV-Rechner %s Anschluss (Port) %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:158
#, python-format
msgid "Using WebDAV directory %s"
msgstr "WebDAV-Verzeichnis %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:189
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "WebDAV erzeugt eine Verbindung unter '%s'"

#: ../duplicity/backends/webdavbackend.py:200
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "WebDAV unbekanntes URI Schema: %s"

#: ../duplicity/backends/webdavbackend.py:219
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "WebDAV %s %s Anfrage mit Headern: %s "

#: ../duplicity/backends/webdavbackend.py:220
#: ../duplicity/backends/webdavbackend.py:243
#, python-format
msgid "WebDAV data length: %s "
msgstr "WebDAV Datenlänge: %s "

#: ../duplicity/backends/webdavbackend.py:223
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "WebDAV Antwortstatus %s mit dem Grund '%s'."

#: ../duplicity/backends/webdavbackend.py:229
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "WebDAV weitersenden an: %s "

#: ../duplicity/backends/webdavbackend.py:231
msgid "WebDAV redirected 10 times. Giving up."
msgstr "WebDAV leitete 10 mal um. Gebe auf."

#: ../duplicity/backends/webdavbackend.py:236
msgid "WebDAV missing location header in redirect response."
msgstr "WebDAV fehlender Lokalisierungs-Header in redirect response."

#: ../duplicity/backends/webdavbackend.py:241
msgid "WebDAV retry request with authentification headers."
msgstr ""
"WebDAV versucht erneut, eine Anforderung mit Authentifizierungs-Headern zu "
"stellen."

#: ../duplicity/backends/webdavbackend.py:242
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "WebDAV %s %s Anfrage2 mit Headern: %s "

#: ../duplicity/backends/webdavbackend.py:246
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "WebDAV Antwort2 Status %s mit Grund »%s«."

#: ../duplicity/backends/webdavbackend.py:263
msgid ""
"python-kerberos needed to use kerberos                           "
"authorization, falling back to basic auth."
msgstr ""
"python-kerberos benötigt Kerberos-Legitimierung, auf die Basislegitimierung "
"wird zurückgegriffen."

#: ../duplicity/backends/webdavbackend.py:267
#, python-format
msgid ""
"Kerberos authorization failed: %s.                          Falling back to "
"basic auth."
msgstr ""
"Kerberos-Legitimierung ist fehlgeschlagen: %s. Auf die Basislegitimierung "
"wird zurückgegriffen."

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr "Fehlendes Verzeichnis »%s« wird erstellt"

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "WebDAV MKCOL %s fehlgeschlagen: %s %s"

#: ../duplicity/backends/webdavbackend.py:382
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "WebDAV Pfad-Dekodierung und Übersetzung: %s -> %s"

#: ../duplicity/backends/webdavbackend.py:427
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr "WebDAV GET Falscher Statuscode %s Grund %s."

#: ../duplicity/backends/webdavbackend.py:449
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "WebDAV PUT Falscher Statuscode %s Grund %s."

#: ../duplicity/backends/webdavbackend.py:469
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL Falscher Statuscode %s Grund %s."

#: ../duplicity/commandline.py:79
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"Use of default filenames is strongly suggested."
msgstr ""
"Achtung: Parameter %s ist veraltet und wird in einer der nächsten Versionen "
"entfernt.\n"
"Bitte verwenden Sie Standarddateinamen."

#: ../duplicity/commandline.py:86
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"--include-filelist and --exclude-filelist now accept globbing characters and "
"should be used instead."
msgstr ""
"Achtung: Option %s ist überflüssig und wird in einer zukünftigen "
"Veröffentlichung entfernt werden.\n"
"--include-filelist und --exclude-filelist erlauben jetzt Platzhalter und "
"sollten stattdessen verwendet werden."

#: ../duplicity/commandline.py:96
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"On many GNU/Linux systems, stdin is represented by /dev/stdin and\n"
"--include-filelist=/dev/stdin or --exclude-filelist=/dev/stdin could\n"
"be used as a substitute."
msgstr ""
"Achtung: Option %s ist überflüssig und wird in einer zukünftigen "
"Veröffentlichung entfernt werden.\n"
"Auf vielen GNU/Linuxsystemen wird die Standardeingabe durch »/dev/stdin« "
"dargestellt und\n"
"--include-filelist=/dev/stdin oder --exclude-filelist=/dev/stdin könnte als "
"Ersatz verwendet werden."

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/commandline.py:257 ../duplicity/commandline.py:267
#: ../duplicity/commandline.py:288 ../duplicity/commandline.py:359
#: ../duplicity/commandline.py:377 ../duplicity/commandline.py:609
#: ../duplicity/commandline.py:642 ../duplicity/commandline.py:855
msgid "path"
msgstr "Pfad"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a hidden GnuPG key. Example:
#. --hidden-encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/commandline.py:283 ../duplicity/commandline.py:290
#: ../duplicity/commandline.py:383 ../duplicity/commandline.py:593
#: ../duplicity/commandline.py:828
msgid "gpg-key-id"
msgstr "GPG-Schlüsselkennung"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/commandline.py:298 ../duplicity/commandline.py:408
#: ../duplicity/commandline.py:878
msgid "shell_pattern"
msgstr "Shell-Ausdruck"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/commandline.py:304 ../duplicity/commandline.py:313
#: ../duplicity/commandline.py:320 ../duplicity/commandline.py:410
#: ../duplicity/commandline.py:417 ../duplicity/commandline.py:430
#: ../duplicity/commandline.py:824
msgid "filename"
msgstr "Dateiname"

#. Used in usage help to represent a regular expression (regexp).
#: ../duplicity/commandline.py:327 ../duplicity/commandline.py:421
msgid "regular_expression"
msgstr "regulärer Ausdruck"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/commandline.py:331 ../duplicity/commandline.py:371
#: ../duplicity/commandline.py:497 ../duplicity/commandline.py:910
msgid "time"
msgstr "Zeit"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:379 ../duplicity/commandline.py:477
#: ../duplicity/commandline.py:500 ../duplicity/commandline.py:601
#: ../duplicity/commandline.py:843
msgid "options"
msgstr "Optionen"

#: ../duplicity/commandline.py:394
#, python-format
msgid ""
"Running in 'ignore errors' mode due to %s; please re-consider if this was "
"not intended"
msgstr ""
"»Fehler vernachlässigen«-Modus läuft gerade, aufgrund von %s; bitte "
"überprüfen wenn das nicht beabsichtigt war"

#. Used in usage help to represent an imap mailbox
#: ../duplicity/commandline.py:406
msgid "imap_mailbox"
msgstr "IMAP-Postfach"

#: ../duplicity/commandline.py:424
msgid "file_descriptor"
msgstr "Dateideskriptor"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/commandline.py:435 ../duplicity/commandline.py:457
#: ../duplicity/commandline.py:474 ../duplicity/commandline.py:483
#: ../duplicity/commandline.py:533 ../duplicity/commandline.py:538
#: ../duplicity/commandline.py:542 ../duplicity/commandline.py:561
#: ../duplicity/commandline.py:567 ../duplicity/commandline.py:571
#: ../duplicity/commandline.py:637 ../duplicity/commandline.py:838
msgid "number"
msgstr "Zahl"

#. Used in usage help (noun)
#: ../duplicity/commandline.py:438
msgid "backup name"
msgstr "Sicherungsname"

#: ../duplicity/commandline.py:556
msgid "policy"
msgstr "Richtlinie"

#: ../duplicity/commandline.py:574
msgid "Hot|Cool|Archive"
msgstr "Hot|Cool|Archiv"

#. noun
#: ../duplicity/commandline.py:577 ../duplicity/commandline.py:580
#: ../duplicity/commandline.py:809
msgid "command"
msgstr "Befehl"

#: ../duplicity/commandline.py:583
msgid "pyrax|cloudfiles"
msgstr "pyrax|cloudfiles"

#: ../duplicity/commandline.py:604
msgid "pem formatted bundle of certificate authorities"
msgstr "PEM-formatiertes Paket von Zertifizierungsstellen"

#: ../duplicity/commandline.py:605
msgid "path to a folder with certificate authority files"
msgstr "Pfad zu einem Verzeichnis mit Dateien der Zertifizierungsstellen"

#. Used in usage help. Example:
#. --timeout <seconds>
#. Used in usage help. Example:
#. --backend-retry-delay <seconds>
#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/commandline.py:614 ../duplicity/commandline.py:648
#: ../duplicity/commandline.py:872
msgid "seconds"
msgstr "Sekunden"

#. abbreviation for "character" (noun)
#: ../duplicity/commandline.py:620 ../duplicity/commandline.py:806
msgid "char"
msgstr "Zeichen"

#: ../duplicity/commandline.py:772
#, python-format
msgid "Using archive dir: %s"
msgstr "Folgendes Archivverzeichnis wird benutzt: %s"

#: ../duplicity/commandline.py:773
#, python-format
msgid "Using backup name: %s"
msgstr "Folgender Sicherungsname wird benutzt: %s"

#: ../duplicity/commandline.py:780
#, python-format
msgid "Command line error: %s"
msgstr "Befehlszeilenfehler: %s"

#: ../duplicity/commandline.py:781
msgid "Enter 'duplicity --help' for help screen."
msgstr "Bitte »--help« aufrufen, um eine Hilfeseite zu erhalten."

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/commandline.py:794
msgid "absolute_path"
msgstr "absoluter_Pfad"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/commandline.py:798
msgid "alias"
msgstr "Alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:803
msgid "bucket_name"
msgstr "Bucket_Name"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/commandline.py:814
msgid "container_name"
msgstr "Containername"

#. noun
#: ../duplicity/commandline.py:817
msgid "count"
msgstr "Anzahl"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/commandline.py:820
msgid "directory"
msgstr "Verzeichnis"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/commandline.py:833
msgid "module"
msgstr "Modul"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:847
msgid "other.host"
msgstr "anderer.Rechner"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:851
msgid "password"
msgstr "Passwort"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:859
msgid "port"
msgstr "Port"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:864
msgid "prefix"
msgstr "Präfix"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/commandline.py:868
msgid "relative_path"
msgstr "relativer_Pfad"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/commandline.py:883
msgid "some_dir"
msgstr "irgendein_Verzeichnis"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:889
msgid "source_dir"
msgstr "Quellverzeichnis"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:894
msgid "source_url"
msgstr "Quelladresse"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:900
msgid "target_dir"
msgstr "Zielverzeichnis"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:905
msgid "target_url"
msgstr "Zieladresse"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:915
msgid "user"
msgstr "Benutzer"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/commandline.py:918
msgid "account_id"
msgstr "Kontokennung"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/commandline.py:922
msgid "application_key"
msgstr "Anwendungsschlüssel"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/commandline.py:926 ../duplicity/dup_collections.py:511
msgid "remote"
msgstr "Entfernt"

#. Header in usage help
#: ../duplicity/commandline.py:945
msgid "Backends and their URL formats:"
msgstr "Hintergrundprogramme und deren Adressformate:"

#. Header in usage help
#: ../duplicity/commandline.py:978
msgid "Commands:"
msgstr "Befehle:"

#: ../duplicity/commandline.py:1003
#, python-format
msgid "Specified archive directory '%s' does not exist, or is not a directory"
msgstr ""
"Das angegebene Archivverzeichnis »%s« besteht nicht oder ist kein Verzeichnis"

#: ../duplicity/commandline.py:1012
#, python-format
msgid ""
"Sign key should be an 8, 16 alt. 40 character hex string, like 'AA0E73D2'.\n"
"Received '%s' instead."
msgstr ""
"Der Signaturschlüssel sollte eine hexadezimale Zahl mit 8, 16 oder 40 "
"Stellen, wie »AA0E73D2« sein.\n"
"Stattdessen wurde »%s« empfangen."

#: ../duplicity/commandline.py:1072
#, python-format
msgid ""
"Restore destination directory %s already exists.\n"
"Will not overwrite."
msgstr ""
"Wiederherstellungszielverzeichnis %s besteht bereits.\n"
"Wird nicht überschrieben."

#: ../duplicity/commandline.py:1077
#, python-format
msgid "Verify directory %s does not exist"
msgstr "Prüfverzeichnis %s besteht nicht"

#: ../duplicity/commandline.py:1083
#, python-format
msgid "Backup source directory %s does not exist."
msgstr "Sicherungsquellverzeichnis %s existiert nicht."

#: ../duplicity/commandline.py:1114
#, python-format
msgid "Command line warning: %s"
msgstr "Befehlszeilenwarnung: %s"

#: ../duplicity/commandline.py:1114
msgid ""
"Selection options --exclude/--include\n"
"currently work only when backing up,not restoring."
msgstr ""
"Die Optionen --exclude/--include funktionieren\n"
"im Moment nur bei der Sicherung, nicht bei der Wiederherstellung."

#: ../duplicity/commandline.py:1150
#, python-format
msgid "GPG binary is %s, version %s"
msgstr "Die GPG Programmdatei ist %s, Version %s"

#: ../duplicity/commandline.py:1177
#, python-format
msgid ""
"Bad URL '%s'.\n"
"Examples of URL strings are \"scp://user@host.net:1234/path\" and\n"
"\"file:///usr/local\".  See the man page for more information."
msgstr ""
"Ungültige Adresse »%s«.\n"
"Beispiele einer Adresszeichenkette sind »scp://benutzer@rechner:1234/pfad« "
"und\n"
"»file:///usr/local«. Für mehr Informationen bitte die man-Seite aufrufen."

#: ../duplicity/commandline.py:1207
msgid "Main action: "
msgstr "Hauptaufgabe: "

#: ../duplicity/diffdir.py:115 ../duplicity/diffdir.py:418
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Fehler %s beim Erhalt des Delta für %s"

#: ../duplicity/diffdir.py:130
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Delta von %s und %s wird erhalten"

#: ../duplicity/diffdir.py:181
#, python-format
msgid "A %s"
msgstr "Ein %s"

#: ../duplicity/diffdir.py:188
#, python-format
msgid "M %s"
msgstr "M %s"

#: ../duplicity/diffdir.py:210
#, python-format
msgid "Comparing %s and %s"
msgstr "%s und %s werden verglichen"

#: ../duplicity/diffdir.py:218
#, python-format
msgid "D %s"
msgstr "D %s"

#: ../duplicity/dup_collections.py:180 ../duplicity/dup_collections.py:194
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "Sicherungssatz löschen: %s fehlt"

#: ../duplicity/dup_collections.py:219
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Kritischer Fehler: Kein Manifest zur letzten Sicherung gefunden"

#: ../duplicity/dup_collections.py:228
msgid ""
"Fatal Error: Remote manifest does not match local one.  Either the remote "
"backup set or the local archive directory has been corrupted."
msgstr ""
"Kritischer Fehler: Manifest im Hintergrundprogramm stimmt nicht mit der "
"lokalen Version überein. Entweder der entfernte Sicherungssatz oder das "
"lokale Archivverzeichnis ist fehlerhaft."

#: ../duplicity/dup_collections.py:236
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr ""
"Kritischer Fehler: Weder das entfernte noch das lokale Manifest ist lesbar."

#: ../duplicity/dup_collections.py:247
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Verarbeitung des lokalen Manifest %s (%s)"

#: ../duplicity/dup_collections.py:259
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "Fehler beim Verarbeiten des entfernten Manifest (%s): %s"

#: ../duplicity/dup_collections.py:262
#, python-format
msgid "Processing remote manifest %s (%s)"
msgstr "Verarbeitung des entfernten Manifest %s (%s)"

#: ../duplicity/dup_collections.py:358
msgid "Preferring Backupset over previous one!"
msgstr "Sicherungssatz wird dem Vorherigen vorgezogen!"

#: ../duplicity/dup_collections.py:361
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr ""
"Schrittweiser Sicherungssatz wird vernachlässigt (Startzeit: %s; benötigt: "
"%s)"

#: ../duplicity/dup_collections.py:366
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr ""
"Schrittweiser Sicherungssatz wurde hinzugefügt (Startzeit: %s; Endzeit: %s)"

#: ../duplicity/dup_collections.py:436
msgid "Chain start time: "
msgstr "Erste Sicherung in dieser Kette: "

#: ../duplicity/dup_collections.py:437
msgid "Chain end time: "
msgstr "Letzte Sicherung in dieser Kette: "

#: ../duplicity/dup_collections.py:438
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Anzahl der enthaltenen Sicherungssätze: %d"

#: ../duplicity/dup_collections.py:440
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Gesamtanzahl der enthaltenen Datenträger: %d"

#: ../duplicity/dup_collections.py:442 ../duplicity/dup_collections.py:1231
msgid "Type of backup set:"
msgstr "Typ des Sicherungssatzes:"

#: ../duplicity/dup_collections.py:442 ../duplicity/dup_collections.py:1231
msgid "Time:"
msgstr "Zeit:"

#: ../duplicity/dup_collections.py:442
msgid "Num volumes:"
msgstr "Datenträgeranz.:"

#: ../duplicity/dup_collections.py:446 ../duplicity/dup_collections.py:1237
msgid "Full"
msgstr "Vollständig"

#: ../duplicity/dup_collections.py:449 ../duplicity/dup_collections.py:1239
msgid "Incremental"
msgstr "Schrittweise"

#: ../duplicity/dup_collections.py:509
msgid "local"
msgstr "Lokal"

#: ../duplicity/dup_collections.py:666
msgid "Collection Status"
msgstr "Sammlungszustand"

#: ../duplicity/dup_collections.py:668
#, python-format
msgid "Connecting with backend: %s"
msgstr "Mit Hintergrundprogramm wird verbunden: %s"

#: ../duplicity/dup_collections.py:670
#, python-format
msgid "Archive dir: %s"
msgstr "Archivverzeichnis: %s"

#: ../duplicity/dup_collections.py:673
#, python-format
msgid "Found %d secondary backup chain."
msgid_plural "Found %d secondary backup chains."
msgstr[0] "%d zusätzliche Sicherungskette gefunden."
msgstr[1] "%d zusätzliche Sicherungsketten gefunden."

#: ../duplicity/dup_collections.py:678
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Zusätzliche Kette %d von %d:"

#: ../duplicity/dup_collections.py:684
msgid "Found primary backup chain with matching signature chain:"
msgstr "Primäre Sicherungskette mit passender Signaturkette gefunden:"

#: ../duplicity/dup_collections.py:688
msgid "No backup chains with active signatures found"
msgstr "Keine Sicherungsketten mit aktiven Signaturen gefunden"

#: ../duplicity/dup_collections.py:691
#, python-format
msgid "Also found %d backup set not part of any chain,"
msgid_plural "Also found %d backup sets not part of any chain,"
msgstr[0] ""
"Zusätzlich wurde %d Sicherungssatz gefunden, der kein Teil einer "
"Sicherungskette ist,"
msgstr[1] ""
"Zusätzlich wurden %d Sicherungssätze gefunden, die kein Teil einer "
"Sicherungskette sind,"

#: ../duplicity/dup_collections.py:695
#, python-format
msgid "and %d incomplete backup set."
msgid_plural "and %d incomplete backup sets."
msgstr[0] "und %d unvollständigen Sicherungssatz."
msgstr[1] "und %d unvollständige Sicherungssätze."

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:700
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr ""
"Sie können diese entfernen, indem Sie duplicity mit dem »cleanup«-Befehl "
"starten."

#: ../duplicity/dup_collections.py:703
msgid "No orphaned or incomplete backup sets found."
msgstr "Keine verwaisten oder unvollständigen Sicherungssätze gefunden."

#: ../duplicity/dup_collections.py:719
#, python-format
msgid "%d file exists on backend"
msgid_plural "%d files exist on backend"
msgstr[0] "%d Datei besteht im Hintergrundprogramm"
msgstr[1] "%d Dateien besteht im Hintergrundprogramm"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] "%d Datei besteht im Puffer"
msgstr[1] "%d Dateien besteht im Puffer"

#: ../duplicity/dup_collections.py:782
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr ""
"Achtung, der letzte Sicherungssatz wird verworfen, weil die Signaturdatei "
"fehlt."

#: ../duplicity/dup_collections.py:805
msgid "Warning, found the following local orphaned signature file:"
msgid_plural "Warning, found the following local orphaned signature files:"
msgstr[0] ""
"Achtung, die folgende verwaiste Signaturdatei wurde im Archivverzeichnis "
"gefunden:"
msgstr[1] ""
"Achtung, die folgenden verwaisten Signaturdateien wurden im "
"Archivverzeichnis gefunden:"

#: ../duplicity/dup_collections.py:814
msgid "Warning, found the following remote orphaned signature file:"
msgid_plural "Warning, found the following remote orphaned signature files:"
msgstr[0] ""
"Achtung, die folgende verwaiste Signaturdatei wurde im Hintergrundprogramm "
"gefunden:"
msgstr[1] ""
"Achtung, die folgenden verwaisten Signaturdateien wurden im "
"Hintergrundprogramm gefunden:"

#: ../duplicity/dup_collections.py:823
msgid "Warning, found signatures but no corresponding backup files"
msgstr ""
"Achtung, es wurden Signaturen ohne dazugehörige Sicherungsdaten gefunden"

#: ../duplicity/dup_collections.py:827
msgid ""
"Warning, found incomplete backup sets, probably left from aborted session"
msgstr ""
"Achtung, es wurden unvollständige Sicherungssätze gefunden, wahrscheinlich "
"von einer abgebrochenen Sitzung zurückgelassen"

#: ../duplicity/dup_collections.py:831
msgid "Warning, found the following orphaned backup file:"
msgid_plural "Warning, found the following orphaned backup files:"
msgstr[0] "Achtung, die folgende Sicherungsdatei ist verwaist:"
msgstr[1] "Achtung, die folgenden Sicherungsdateien sind verwaist:"

#: ../duplicity/dup_collections.py:848
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Sicherungsketten aus der Liste der Dateien werden entnommen: %s"

#: ../duplicity/dup_collections.py:860
#, python-format
msgid "File %s is part of known set"
msgstr "Datei %s ist Teil eines bekannten Satzes"

#: ../duplicity/dup_collections.py:863
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr ""
"Datei %s ist nicht Teil eines bekannten Sicherungssatzes; neuer Satz wird "
"erstellt"

#: ../duplicity/dup_collections.py:868
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Datei »%s« wird vernachlässigt (vom Sicherungssatz abgelehnt)"

#: ../duplicity/dup_collections.py:884
#, python-format
msgid "Found backup chain %s"
msgstr "Sicherungskette %s gefunden"

#: ../duplicity/dup_collections.py:889
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Satz %s zur bestehenden Sicherungskette %s hinzugefügt"

#: ../duplicity/dup_collections.py:893
#, python-format
msgid "Found orphaned set %s"
msgstr "Verwaister Satz %s gefunden"

#: ../duplicity/dup_collections.py:1043
#, python-format
msgid ""
"No signature chain for the requested time. Using oldest available chain, "
"starting at time %s."
msgstr ""
"Keine Signaturkette für die angeforderte Zeit. Nutze die älteste verfügbare "
"Kette, starte bei der Zeit %s."

#: ../duplicity/dup_collections.py:1229
#, python-format
msgid "File: %s"
msgstr "Datei: %s"

#: ../duplicity/dup_collections.py:1230
#, python-format
msgid "Total number of backup: %d"
msgstr "Gesamtzahl an Sicherungen: %d"

#: ../duplicity/dup_collections.py:1231
msgid "Type of file change:"
msgstr "Art der Dateiänderung:"

#: ../duplicity/dup_main.py:121
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Eingestellte PASSPHRASE als SIGN_PASSPHRASE benutzen"

#: ../duplicity/dup_main.py:128
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Eingestellte SIGN_PASSPHRASE als PASSPHRASE benutzen"

#: ../duplicity/dup_main.py:167
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE nicht eingestellt, Benutzer wird gefragt."

#: ../duplicity/dup_main.py:182
msgid "GnuPG passphrase for signing key:"
msgstr "GnuPG-Passphrase für die Schlüsselsignierung:"

#: ../duplicity/dup_main.py:187
msgid "GnuPG passphrase:"
msgstr "GnuPG-Passphrase:"

#: ../duplicity/dup_main.py:192
msgid "Retype passphrase for signing key to confirm: "
msgstr ""
"Zur Bestätigung bitte die Passphrase für die Schlüsselsignierung "
"wiederholen: "

#: ../duplicity/dup_main.py:194
msgid "Retype passphrase to confirm: "
msgstr "Zur Bestätigung bitte die Passphrase wiederholen: "

#: ../duplicity/dup_main.py:197
msgid "First and second passphrases do not match!  Please try again."
msgstr ""
"Erste und zweite Passphrase stimmen nicht überein! Bitte nochmal versuchen."

#: ../duplicity/dup_main.py:204
msgid ""
"Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr ""
"Bei symmetrischer Verschlüsselung darf keine leere Passphrase benutzt "
"werden! Bitte nochmal versuchen."

#: ../duplicity/dup_main.py:261
#, python-format
msgid ""
"File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Datei %s ist komplett im Sicherungssatz.\n"
"Nach Neustart erfolgt Fortsetzung mit Datei %s."

#: ../duplicity/dup_main.py:270
#, python-format
msgid ""
"File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Datei %s fehlt im Sicherungssatz.\n"
"Beim Neustart wird fortgefahren mit Datei %s."

#: ../duplicity/dup_main.py:319
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Datei %s wurde während des Hochladens beschädigt."

#: ../duplicity/dup_main.py:359
msgid ""
"Restarting backup, but current encryption settings do not match original "
"settings"
msgstr ""
"Datensicherung wird neu gestartet, die aktuellen "
"Verschlüsselungseinstellungen entsprechen jedoch nicht den ursprünglichen "
"Einstellungen"

#: ../duplicity/dup_main.py:382
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Neustart nach Datenträger %s, Datei %s, Block %s"

#: ../duplicity/dup_main.py:452
#, python-format
msgid "Processed volume %d"
msgstr "Datenträger %d wurde verarbeitet"

#: ../duplicity/dup_main.py:602
msgid ""
"Fatal Error: Unable to start incremental backup.  Old signatures not found "
"and incremental specified"
msgstr ""
"Schwerwiegender Fehler: schrittweise Sicherung konnte nicht gestartet "
"werden. Alte Signaturen nicht gefunden und schrittweise Sicherung ausgewählt."

#: ../duplicity/dup_main.py:606
msgid "No signatures found, switching to full backup."
msgstr "Keine Signaturen gefunden, vollständige Sicherung wird durchgeführt."

#: ../duplicity/dup_main.py:620
msgid "Backup Statistics"
msgstr "Sicherungsstatistiken"

#: ../duplicity/dup_main.py:726
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s wurde nicht im Archiv gefunden - keine Dateien wiederhergestellt."

#: ../duplicity/dup_main.py:730
msgid "No files found in archive - nothing restored."
msgstr "Keine Dateien im Archiv - nichts wiederhergestellt."

#: ../duplicity/dup_main.py:763
#, python-format
msgid "Processed volume %d of %d"
msgstr "Datenträger %d von %d wurde verarbeitet"

#: ../duplicity/dup_main.py:797
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Ungültige Daten – %s-Hash stimmt nicht für die Datei:"

#: ../duplicity/dup_main.py:800
#, python-format
msgid "Calculated hash: %s"
msgstr "Erstellter Hash: %s"

#: ../duplicity/dup_main.py:801
#, python-format
msgid "Manifest hash: %s"
msgstr "Manifest-Hashcode: %s"

#: ../duplicity/dup_main.py:844
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Datenträger wurde von Schlüssel %s, anstatt von %s unterschrieben"

#: ../duplicity/dup_main.py:876
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Überprüfung vollständig: %s, %s."

#: ../duplicity/dup_main.py:877
#, python-format
msgid "%d file compared"
msgid_plural "%d files compared"
msgstr[0] "%d Datei verglichen"
msgstr[1] "%d Dateien verglichen"

#: ../duplicity/dup_main.py:879
#, python-format
msgid "%d difference found"
msgid_plural "%d differences found"
msgstr[0] "%d Unterschied gefunden"
msgstr[1] "%d Unterschiede gefunden"

#: ../duplicity/dup_main.py:898
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Keine überflüssigen Dateien gefunden, es wurde nichts gelöscht."

#: ../duplicity/dup_main.py:903
msgid "Deleting this file from backend:"
msgid_plural "Deleting these files from backend:"
msgstr[0] "Diese Datei wird vom Hintergrundprogramm gelöscht:"
msgstr[1] "Diese Dateien wird vom Hintergrundprogramm gelöscht:"

#: ../duplicity/dup_main.py:914
msgid "Found the following file to delete:"
msgid_plural "Found the following files to delete:"
msgstr[0] "Folgende Datei zum Löschen gefunden:"
msgstr[1] "Folgende Dateien zum Löschen gefunden:"

#: ../duplicity/dup_main.py:917
msgid "Run duplicity again with the --force option to actually delete."
msgstr ""
"Bitte duplicity noch einmal mit der »--force«-Option ausführen, um "
"tatsächlich zu löschen."

#: ../duplicity/dup_main.py:960
msgid "There are backup set(s) at time(s):"
msgstr "Es bestehen Sicherungssätze zu folgenden Zeitpunkten:"

#: ../duplicity/dup_main.py:962
msgid "Which can't be deleted because newer sets depend on them."
msgstr ""
"Welche nicht gelöscht werden können, weil neuere Sicherungssätze davon "
"abhängen."

#: ../duplicity/dup_main.py:966
msgid ""
"Current active backup chain is older than specified time.  However, it will "
"not be deleted.  To remove all your backups, manually purge the repository."
msgstr ""
"Aktuell aktive Sicherungskette ist älter als der angegebene Zeitpunkt. Sie "
"wird nicht gelöscht. Um alle Sicherungen zu entfernen, bitte die Paketquelle "
"manuell löschen."

#: ../duplicity/dup_main.py:979
msgid "No old backup sets found, nothing deleted."
msgstr "Keine alten Sicherungssätze gefunden, nichts gelöscht."

#: ../duplicity/dup_main.py:982
msgid "Deleting backup chain at time:"
msgid_plural "Deleting backup chains at times:"
msgstr[0] "Sicherungskette dieses Zeitpunkts wird gelöscht:"
msgstr[1] "Sicherungskette dieser Zeitpunkte wird gelöscht:"

#: ../duplicity/dup_main.py:994
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "Jede schrittweise Signaturkette die bei %s verankert ist wird gelöscht"

#: ../duplicity/dup_main.py:996
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr ""
"Jede schrittweise Sicherungskette die bei %s verankert ist wird gelöscht"

#: ../duplicity/dup_main.py:999
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Vollständige Signaturkette wird gelöscht: %s"

#: ../duplicity/dup_main.py:1001
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Vollständige Sicherungskette wird gelöscht: %s"

#: ../duplicity/dup_main.py:1007
msgid "Found old backup chain at the following time:"
msgid_plural "Found old backup chains at the following times:"
msgstr[0] "Alte Sicherungskette zu folgendem Zeitpunkt gefunden:"
msgstr[1] "Alte Sicherungeketten zu folgenden Zeitpunkten gefunden:"

#: ../duplicity/dup_main.py:1011
msgid "Rerun command with --force option to actually delete."
msgstr ""
"Bitte Befehl erneut mit »--force«-Option ausführen, um tatsächlich zu "
"löschen."

#: ../duplicity/dup_main.py:1034
msgid "No old backup sets found."
msgstr "Keine alten Backup-Sets gefunden."

#: ../duplicity/dup_main.py:1049
#, python-format
msgid "Signature %s already replicated"
msgstr "Signatur %s bereits repliziert"

#: ../duplicity/dup_main.py:1056 ../duplicity/dup_main.py:1095
#, python-format
msgid "Replicating %s."
msgstr "Replizieren von %s."

#: ../duplicity/dup_main.py:1083
#, python-format
msgid "Backupset %s already replicated"
msgstr "Backup-Set %s bereits repliziert"

#: ../duplicity/dup_main.py:1231
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Lokal %s wird gelöscht (keine Auswirkungen am Hintergrundprogramm)."

#: ../duplicity/dup_main.py:1236
#, python-format
msgid "Unable to delete %s: %s"
msgstr "%s kann nicht gelöscht werden: %s"

#: ../duplicity/dup_main.py:1270 ../duplicity/dup_temp.py:271
#, python-format
msgid "Failed to read %s: %s"
msgstr "%s konnte nicht gelesen werden: %s"

#: ../duplicity/dup_main.py:1284
#, python-format
msgid "Copying %s to local cache."
msgstr "%s wird zum lokalen Puffer kopiert."

#: ../duplicity/dup_main.py:1332
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr ""
"Lokale und entfernte Metadaten sind bereits synchron, kein Abgleich benötigt."

#: ../duplicity/dup_main.py:1337
msgid "Synchronizing remote metadata to local cache..."
msgstr "Entfernte Metadaten werden zum lokalen Puffer synchronisiert …"

#: ../duplicity/dup_main.py:1350
msgid "Sync would copy the following from remote to local:"
msgstr ""
"Synchronisation würde die folgenden Daten vom entfernten Rechner hierher "
"kopieren:"

#: ../duplicity/dup_main.py:1353
msgid "Sync would remove the following spurious local files:"
msgstr ""
"Synchronisation würde die folgenden überflüssigen lokalen Dateien entfernen:"

#: ../duplicity/dup_main.py:1397
msgid "Unable to get free space on temp."
msgstr "Der temporäre Speicherplatz ist belegt."

#: ../duplicity/dup_main.py:1405
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr ""
"%d temporärer Speicherplatz ist verfügbar, die Sicherung benötigt ungefähr "
"%d."

#: ../duplicity/dup_main.py:1408
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr ""
"%d temporärer Speicherplatz ist verfügbar, die Sicherung wird ungefähr %d "
"nutzen."

#: ../duplicity/dup_main.py:1416
msgid "Unable to get max open files."
msgstr "Der Höchstwert für offene Dateien konnte nicht festgestellt werden."

#: ../duplicity/dup_main.py:1420
#, python-format
msgid ""
"Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr ""
"Limit für gleichzeitig offene Dateien von %s ist zu niedrig, sollte >=1024 "
"sein.\n"
"Bitte »ulimit -n 1024« ausführen, um dieses zu berichtigen.\n"

#: ../duplicity/dup_main.py:1472
msgid ""
"RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr ""
"NEUSTART: Das Hochladen des ersten Datenträgers ist fehlgeschlagen.\n"
"         Neustart nicht möglich, Sicherung wird vom Anfang wiederaufgenommen."

#: ../duplicity/dup_main.py:1478
#, python-format
msgid ""
"RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr ""
"NEUSTART: Das Hochladen der Datenträger %d bis %d ist fehlgeschlagen.\n"
"        Sicherung wird mit Datenträger %d wiederaufgenommen."

#: ../duplicity/dup_main.py:1485
#, python-format
msgid ""
"RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last "
"partial\n"
"         backup then restart the backup from the beginning."
msgstr ""
"NEUSTART: Unmöglicher Sicherungszustand:\n"
"Manifest hat eine Größe von %d, Hintergrundprogramm hat eine Größe von %d.\n"
"        Neustart nicht möglich … duplicity wird die letzte unvollständige\n"
"        Sicherung entfernen und von vorn anfangen."

#: ../duplicity/dup_main.py:1506
msgid ""
"\n"
"PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n"
"\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""
"\n"
"Die Umgebungsvariable PYTHONOPTIMIZE bewirkt,\n"
"dass duplicity seine eigenen Sicherungen nicht wiedererkennt.\n"
"Bitte entfernen Sie PYTHONOPTIMIZE aus Ihrer Umgebung\n"
"und starten Sie die Sicherung neu.\n"
"\n"
"Siehe https://bugs.launchpad.net/duplicity/+bug/931175\n"

#: ../duplicity/dup_main.py:1529
#, python-format
msgid "Acquiring lockfile %s"
msgstr "Die Sperrdatei %s wird erfasst"

#: ../duplicity/dup_main.py:1587
#, python-format
msgid "Last %s backup left a partial set, restarting."
msgstr ""
"Das letzte %s-Sicherung hinterließ einen Teil eines Sicherungssatzes, "
"Neustart wird durchgeführt."

#: ../duplicity/dup_main.py:1591
#, python-format
msgid "Cleaning up previous partial %s backup set, restarting."
msgstr ""
"Der letzte unvollständige %s-Sicherungssatz wird entfernt, Neustart wird "
"durchgeführt."

#: ../duplicity/dup_main.py:1603
msgid "Last full backup date:"
msgstr "Letzte vollständige Sicherung:"

#: ../duplicity/dup_main.py:1605
msgid "Last full backup date: none"
msgstr "Letzte vollständige Sicherung: Keine"

#: ../duplicity/dup_main.py:1608
msgid "Last full backup is too old, forcing full backup"
msgstr ""
"Letzte vollständige Sicherung ist zu alt, vollständige Sicherung wird "
"erzwungen"

#: ../duplicity/dup_main.py:1657
msgid ""
"When using symmetric encryption, the signing passphrase must equal the "
"encryption passphrase."
msgstr ""
"Bei symmetrischer Verschlüsselung müssen die Passphrasen für Signatur und "
"Verschlüsselung übereinstimmen."

#: ../duplicity/dup_time.py:63
#, python-format
msgid ""
"Bad interval string \"%s\"\n"
"\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr ""
"Ungültige Intervallzeichenkette »%s«\n"
"\n"
"Intervalle sind angegeben wie 2Y (2 Jahre) oder 2h30m (2,5 Stunden).\n"
"Die erlaubten Spezialzeichen sind s, m, h, D, W, M, und Y.\n"
"Weitere Informationen sind im Handbuch zu finden."

#: ../duplicity/dup_time.py:69
#, python-format
msgid ""
"Bad time string \"%s\"\n"
"\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr ""
"Ungültige Zeitzeichenkette »%s«\n"
"\n"
"Gültige Zeitzeichenkette sind Intervalle (wie »3D46s«), w3-Datum-"
"Zeitzeichenketten,\n"
"wie »2002-04-26T04:22:01-07:00« (Zeichenketten wie »2002-04-26T04:22:01« "
"sind\n"
"auch gültig - duplicity wird die aktuelle Zeitzone nutzen, oder gewöhnliche "
"Daten\n"
"wie 2/4/1997 oder 2001-04-23 (verschiedene Kombinationen sind gültig, aber "
"vor\n"
"dem Monat kommt immer der Tag)."

#: ../duplicity/gpginterface.py:238
msgid "Threading not available -- zombie processes may appear"
msgstr "Einfädelung ist nicht verfügbar – Zombieprozesse können auftreten."

#: ../duplicity/gpginterface.py:705
#, python-format
msgid "GPG process %d terminated before wait()"
msgstr "GPG-Prozess %d beendet vor »wait()«."

#: ../duplicity/lazy.py:352
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Achtung: Alt-Index %s >= Neu-Index %s"

#: ../duplicity/lazy.py:427
#, python-format
msgid "Error '%s' processing %s"
msgstr "Fehler »%s« beim Verarbeiten von %s"

#: ../duplicity/lazy.py:437
#, python-format
msgid "Skipping %s because of previous error"
msgstr "%s wird wegen vorherigem Fehler übersprungen"

#: ../duplicity/librsync.py:195
msgid ""
"basis_file must be a (true) file or an object whose file attribute is the "
"underlying true file object"
msgstr ""
"basis_file muss eine (echte) Datei oder ein Objekt sein, dessen Datei-"
"Attribut das zugrundeliegende Dateiobjekt ist"

#: ../duplicity/manifest.py:96
#, python-format
msgid ""
"Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr ""
"Kritischer Fehler: Sicherungsquellrechner hat sich geändert.\n"
"Aktueller Rechnername: %s\n"
"Vorheriger Rechnername: %s"

#: ../duplicity/manifest.py:103
#, python-format
msgid ""
"Fatal Error: Backup source directory has changed.\n"
"Current directory: %s\n"
"Previous directory: %s"
msgstr ""
"Kritischer Fehler: Sicherungsquellverzeichnis hat sich geändert.\n"
"Aktuelles Verzeichnis: %s\n"
"Vorheriges Verzeichnis: %s"

#: ../duplicity/manifest.py:113
msgid ""
"Aborting because you may have accidentally tried to backup two different "
"data sets to the same remote location, or using the same archive directory.  "
"If this is not a mistake, use the --allow-source-mismatch switch to avoid "
"seeing this message"
msgstr ""
"Abbruch, weil Sie möglicherweise versehentlich versucht haben, zwei "
"verschiedene Datensätze zum gleichen entfernten Ort oder zum selben "
"Archivverzeichnis zu sichern. Wenn dies kein Fehler ist, verwenden Sie den "
"»allow-source-mismatch«-Parameter, um diese Meldung auszuschalten."

#: ../duplicity/manifest.py:215
#, python-format
msgid "Found manifest volume %s"
msgstr "Manifest-Volumen %s gefunden"

#: ../duplicity/manifest.py:222
#, python-format
msgid "Found %s volumes in manifest"
msgstr "%s Volumen in Manifest gefunden"

#: ../duplicity/manifest.py:239
#, python-format
msgid ""
"Manifest file '%s' is corrupt: File count says %d, File list contains %d"
msgstr ""
"Die Manifestdatei '%s' ist beschädigt: Dateianzahl ist %d, Dateiliste "
"enthält %d"

#: ../duplicity/manifest.py:258
msgid "Manifests not equal because different volume numbers"
msgstr "Manifests sind ungleich aufgrund verschiedener Datenträgerzahlen"

#: ../duplicity/manifest.py:263
msgid "Manifests not equal because volume lists differ"
msgstr "Manifests sind ungleich aufgrund verschiedenen Datenträgerlisten"

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because hosts or directories differ"
msgstr ""
"Manifests sind ungleich aufgrund Rechner- oder Verzeichnisunterschieden"

#: ../duplicity/manifest.py:425
msgid "Warning, found extra Volume identifier"
msgstr "Achtung, zusätzliche Datenträgerbezeichnung gefunden"

#: ../duplicity/manifest.py:451
msgid "Other is not VolumeInfo"
msgstr "Anderes ist keine Datenträgerinfo"

#: ../duplicity/manifest.py:454
msgid "Volume numbers don't match"
msgstr "Datenträgerzahlen stimmen nicht überein"

#: ../duplicity/manifest.py:457
msgid "start_indicies don't match"
msgstr "Startindizes stimmen nicht überein"

#: ../duplicity/manifest.py:460
msgid "end_index don't match"
msgstr "Endindex stimmt nicht überein"

#: ../duplicity/manifest.py:467
msgid "Hashes don't match"
msgstr "Prüfsummen stimmen nicht überein"

#: ../duplicity/patchdir.py:84 ../duplicity/patchdir.py:89
#, python-format
msgid "Patching %s"
msgstr "%s wird repariert"

#: ../duplicity/patchdir.py:550
#, python-format
msgid "Error '%s' patching %s"
msgstr "Fehler »%s« beim Reparieren von %s"

#: ../duplicity/patchdir.py:625
#, python-format
msgid "Writing %s of type %s"
msgstr "Schreibvorgang von %s des Typs %s"

#: ../duplicity/path.py:116
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr ""
"Achtung: %s ungültige Gerätenummern (0x%X), diese werden wie (0, 0) "
"behandelt."

#: ../duplicity/path.py:245 ../duplicity/path.py:306
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Achtung: %s hat eine negative mtime, diese wird wie 0 behandelt."

#: ../duplicity/path.py:370
msgid "Difference found:"
msgstr "Unterschiede gefunden:"

#: ../duplicity/path.py:379
#, python-format
msgid "New file %s"
msgstr "Neue Datei %s"

#: ../duplicity/path.py:382
#, python-format
msgid "File %s is missing"
msgstr "Datei %s fehlt"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Datei %%s ist vom Typ %s, erwartet wurde %s"

#: ../duplicity/path.py:391 ../duplicity/path.py:417
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Datei %%s hat die Zugriffsrechte %s, erwartet wurde %s"

#: ../duplicity/path.py:396
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Datei %%s hat mtime %s, erwartet wurde %s"

#: ../duplicity/path.py:404
#, python-format
msgid "Data for file %s is different"
msgstr "Daten für Datei %s sind unterschiedlich"

#: ../duplicity/path.py:412
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Symbolischer Verweis %%s zeigt auf %s, erwartet wurde %s"

#: ../duplicity/path.py:421
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Gerätedatei %%s hat Nummern %s, erwartet wurde %s"

#: ../duplicity/path.py:603
#, python-format
msgid "Making directory %s"
msgstr "Verzeichnis %s wird erstellt"

#: ../duplicity/path.py:613
#, python-format
msgid "Deleting %s"
msgstr "»%s« wird gelöscht"

#: ../duplicity/path.py:622
#, python-format
msgid "Touching %s"
msgstr "Zugriffszeit %s wird erstellt oder aktualisiert"

#: ../duplicity/path.py:629
#, python-format
msgid "Deleting tree %s"
msgstr "Baumstruktur %s wird gelöscht"

#: ../duplicity/robust.py:64
#, python-format
msgid "Error listing directory %s"
msgstr "Fehler beim Auflisten des Verzeichnisses %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Skipping socket %s"
msgstr "Socket %s wird übersprungen"

#: ../duplicity/selection.py:125
#, python-format
msgid "Error initializing file %s"
msgstr "Fehler beim Initialisieren der Datei %s"

#: ../duplicity/selection.py:129 ../duplicity/selection.py:154
#: ../duplicity/selection.py:454
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Fehler beim Zugriff auf möglicherweise gesperrte Datei %s"

#: ../duplicity/selection.py:169
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Achtung: Basis %s besteht nicht, es wird fortgefahren"

#: ../duplicity/selection.py:172 ../duplicity/selection.py:190
#: ../duplicity/selection.py:193
#, python-format
msgid "Selecting %s"
msgstr "%s wird ausgewählt"

#: ../duplicity/selection.py:290
#, python-format
msgid ""
"Fatal Error: The file specification\n"
"    %s\n"
"cannot match any files in the base directory\n"
"    %s\n"
"Useful file specifications begin with the base directory or some\n"
"pattern (such as '**') which matches the base directory."
msgstr ""
"Kritischer Fehler: Die Dateiangaben\n"
"    %s\n"
"können nicht mit Dateien im Basisverzeichnis\n"
"    %s \n"
"übereinstimmen. Nützliche Dateiangaben beginnen mit dem Basisverzeichnis "
"oder einem\n"
"Muster (wie »**«), welches mit dem Basisverzeichnis übereinstimmt."

#: ../duplicity/selection.py:299
#, python-format
msgid ""
"Fatal Error while processing expression\n"
"%s"
msgstr ""
"Kritischer Fehler beim Verarbeiten des Ausdrucks\n"
"%s"

#: ../duplicity/selection.py:308
#, python-format
msgid ""
"Last selection expression:\n"
"    %s\n"
"only specifies that files be included.  Because the default is to\n"
"include all files, the expression is redundant.  Exiting because this\n"
"probably isn't what you meant."
msgstr ""
"Letzter Auswahlausdruck:\n"
"    %s\n"
"Gibt nur Dateien an, die einbezogen werden. Weil standardmäßig\n"
"alle Dateien einbezogen werden ist der Ausdruck überflüssig.\n"
"Wird beendet, weil das wahrscheinlich nicht das ist, was Sie beabsichtigten."

#: ../duplicity/selection.py:364
#, python-format
msgid "Reading globbing filelist %s"
msgstr "Ausdruckbasierte Dateiliste wird gelesen %s"

#: ../duplicity/selection.py:397
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Fehler beim Auswerten des regulären Ausdrucks %s"

#: ../duplicity/tempdir.py:160
#, python-format
msgid "Using temporary directory %s"
msgstr "Temporäres Verzeichnis %s wird benutzt"

#: ../duplicity/tempdir.py:204
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "(mktemp) temporäre Datei %s wird registriert"

#: ../duplicity/tempdir.py:226
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "(mkstemp) temporäre Datei %s wird registriert"

#: ../duplicity/tempdir.py:258
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Temporäre Datei %s wird vergessen"

#: ../duplicity/tempdir.py:261
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr ""
"Unbekannte temporäre Datei %s wird versucht zu vergessen - das ist "
"wahrscheinlich ein Fehler."

#: ../duplicity/tempdir.py:280
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Bereits gemerkte temporäre Datei %s wird entfernt"

#: ../duplicity/tempdir.py:283
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Entfernen der temporären Datei %s ist fehlgeschlagen"

#: ../duplicity/tempdir.py:288
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr ""
"Aufräumen des temporären Verzeichnisses %s fehlgeschlagen - das ist "
"wahrscheinlich ein Fehler."

#: ../duplicity/util.py:144
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr ""
"VERNACHLÄSSIGTER FEHLER: Achtung: Fehler wird wie verlangt vernachlässigt: "
"%s: %s"

#: ../duplicity/util.py:211
#, python-format
msgid "Releasing lockfile %s"
msgstr "Sperrdatei »%s« wird freigegeben"

#~ msgid "MultiBackend: list from %s: %s"
#~ msgstr "MehrfachHintergrundprogramm: Liste von %s: %s"

#~ msgid ""
#~ "Warning: exclude-device-files is not the first selector.\n"
#~ "This may not be what you intended"
#~ msgstr ""
#~ "Achtung: »exclude-device-files« ist nicht die erste Auswahl.\n"
#~ "Das ist möglicherweise nicht, was Sie beabsichtigen."

#~ msgid ""
#~ "For certificate verification with python 2.7.8 or earlier a cacert "
#~ "database\n"
#~ "file is needed in one of these locations: %s\n"
#~ "Hints:\n"
#~ "  Consult the man page, chapter 'SSL Certificate Verification'.\n"
#~ "  Consider using the options --ssl-cacert-file, --ssl-no-check-"
#~ "certificate ."
#~ msgstr ""
#~ "Für zertifizierte Verifikation mit Phyton 2.7.8 oder früher ist eine "
#~ "Cacert Datenbankdatei\n"
#~ "an einer dieser Stellen notwendig: %s\n"
#~ "Hinweise:\n"
#~ "  Lesen Sie die Hilfeseiten, im Kapitel  'SSL Certificate Verification'.\n"
#~ "  Prüfen Sie die Optionen --ssl-cacert-file, --ssl-no-check-certificate ."
